### PR TITLE
Music player

### PR DESCRIPTION
**Pull request by jjamaltwin**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant
-
- [ ] New feature (non-breaking change which adds functionality)

## Change Status

- [ ] Complete, but not tested (may need new tests)
# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: groovin-music/src/components/MusicPlayer/MusicPlayerContext.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
+import React, { useState } from 'react';
+
+
+
+const MusicPlayerContext = React.createContext([{}, () => {}]);
+
+const MusicPlayerProvider = (props) => {
+  const [state, setState] = useState({
```
Comment (djzaragoza) [inline groovin-music/src/components/MusicPlayer/MusicPlayerContext.js]: Hooks!

File: groovin-music/src/components/MusicPlayer/PlayerControls.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import useMusicPlayer from '../hooks/useMusicPlayer';
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import {faPause, faPlay, faStepBackward, faStepForward} from '@fortawesome/free-solid-svg-icons';
+
+function PlayerControls() {
+    const {isPlaying, currentTrackName, togglePlay, playPreviousTrack, playNextTrack} = useMusicPlayer();
```
Comment (djzaragoza) [inline groovin-music/src/components/MusicPlayer/PlayerControls.js]: nicely done here

**Review by djzaragoza — APPROVED** [review]
overall this is awesome, nice work!

**Review by Pherpher089-zz — COMMENTED** [review]
All looks good

**Pull request merged**